src/Controller/ArticleController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Article;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+class ArticleController extends AbstractController
+{
+    /**
+     * @Route("/", name="homepage")
+     */
+    public function listArticle()
+    {
+        $articles = $this->getDoctrine()
+            ->getRepository(Article::class)
+            ->findAll();
+
```

Comment:
findAll() method is not the best approach, but it is good for now
Better to use some pagination and make repository method that queries exactly what you need